packages/client/src/App.re

```diff
@@ -18,7 +20,7 @@ let make = () => {
       | ["boka-resa"] => <Book />
       | ["resor"] => <Trips />
       | ["resa", id] => <Trip id />
-      | _ => <p> {I18n.Error.toString(FourOFour)->React.string} </p>
+      | _ => <Typography.Error> {errorToString(FourOFour)} </Typography.Error>
```

Comment:
I like this! I wonder if we could take it one step further and only pass it a variant? `FourOFour`, it would save us from having to `LocaleContext.use` everywhere. Might be something for another ticket!

Reply:
Sweet lets do it. I'll have a look while I'm solving merge conflicts

Reply:
Lets save this for later. Need to think it through a bit more.